critiquebrainz/frontend/views/recording.py

```diff
@@ -0,0 +1,53 @@
+from flask import Blueprint, render_template, request
+from flask_login import current_user
+from werkzeug.exceptions import NotFound
+from flask_babel import gettext
+import critiquebrainz.db.review as db_review
+import critiquebrainz.frontend.external.musicbrainz_db.recording as mb_recording
+import critiquebrainz.frontend.external.musicbrainz_db.exceptions as mb_exceptions
+from critiquebrainz.frontend.forms.rate import RatingEditForm
+from critiquebrainz.frontend.views import get_avg_rating
+
+
+recording_bp = Blueprint('recording', __name__)
+
+
+@recording_bp.route('/<uuid:id>')
+def entity(id):
+    id = str(id)
+    try:
+        recording = mb_recording.get_recording_by_id(id)
+    except mb_exceptions.NoDataFoundException:
+        raise NotFound(gettext("Sorry, we couldn't find a recording with that MusicBrainz ID."))
+
+    if 'url-rels' in recording:
+        external_reviews = list(filter(lambda rel: rel['type'] == 'review', recording['url-rels']))
+    else:
+        external_reviews = []
+
+    limit = int(request.args.get('limit', default=10))
```

Comment:
Could you also move the default to a config value?

Reply:
Updated.